rust_dev_preview/examples/ses/src/bin/send-email.rs

```diff
@@ -55,12 +55,13 @@ async fn send_message(
 
     let contacts = resp.contacts().unwrap_or_default();
 
-    let cs: String = contacts
+    let cs: Vec<String> = contacts
```

Comment:
I would do something like:
```rust
let mut builder = Destination::builder();
for contact in contacts {
  builder.to_addresses(contact.email_address().unwrap());
}
let dest = builder.build();
```

Reply:
I generally agree but I'm not sure how consistent we've been on this, so I'll approve this now & do a pass later to clean it up across the board.